README.rst

```diff
@@ -87,4 +87,110 @@ the exporting file and override ``get_sharing_configs_export_data`` method
+        secondary = models.CharField("secondary", max_length=60)
+        accent = models.CharField("accent", max_length=60)
+        primary_fg = models.CharField("primary_fg", max_length=60)
+        
+        
+Register the ``Theme`` model in the admin and include our two mixins to introduce the UI to import and export objects, in this case, themes.
+Sharing Configs does not know how to import or export your model, so you will need to write this yourself. You 
+can override the methods introduced by the mixins: ``get_sharing_configs_export_data`` and ``get_sharing_configs_import_data``
+
+.. code-block:: python
+    # admin.py
+    import json
+
+    from django.contrib import admin
+    from django.forms.models import model_to_dict
+    from django.shortcuts import get_object_or_404
+
+    from sharing_configs.admin import SharingConfigsExportMixin, SharingConfigsImportMixin
+    from sharing_configs.forms import ExportToForm, ImportForm
+
+    from .models import Configuration, Theme
+
+
+    class ThemeAdmin(SharingConfigsExportMixin, SharingConfigsImportMixin, admin.ModelAdmin):        
+
+        def get_sharing_configs_export_data(self, obj: object) -> bytes:
+            """convert the theme to JSON."""
+            theme = get_object_or_404(Theme, id=obj.id)
+            theme_dict = model_to_dict(theme)
+            theme_dict.pop("id", None)
+            dump_json_theme = json.dumps(cleaned_theme_dict, sort_keys=True, default=str)        
+            return dump_json_theme.encode("utf-8")
+
+        def get_sharing_configs_import_data(self, content: bytes) -> object:
+            """
+            Convert JSON to a new theme instance. Typically, the JSON that is read here is the same as that 
+            the JSON generated by the above function.
+            """              
+            decoded_content = content.decode("utf-8")
+            theme_dict = json.loads(decoded_content)        
+            return ColorTheme.objects.create(**theme_dict)       
+
+
+That takes care of the import and export functionality for exchaning color-themes. To make it actually working, 
+we complete this example with some additional code.
+Create a ``context_processors.py`` file, to pass the currently configured theme to the template context:
+
+.. code-block:: python
+
+    def set_admin_color(request:object)->dict:
+        """
+        create a dictionary of color variables to pass to the base_site.html Django admin page
+        """
+        conf = Configuration.get_solo()
+        theme = conf.theme
+        if theme is None:
+            return {}
+
+        return {
+            "primary": theme.primary,
+            "secondary": theme.secondary,            
+            "primary_fg": theme.primary_fg,
+            "accent": theme.accent
+            
+        }
+
+Finally, pass the theme context variables to an overriden ``base_site.html`` in the templates folder.
+
+.. code-block::
+  {% extends "admin/base_site.html" %}
```

Comment:
This code block doesnt seem to render in markdown. Maybe indent is wrong or it needs a return after the block statement.